### PR TITLE
CSU-291: New student dashboard

### PR DESCRIPTION
**Pull request by AnaBZC**
## ID Ticket: 
- [CSU-291](https://app.clickup.com/t/36718269/CSU-291)

### Description
- Create new component CTA card

## Steps
- Go to [CTA card ](http://localhost:6006/?path=/story/components-cta-card--default)
- Check that it looks like the [design](https://www.figma.com/file/Lh3hcoB0HXskHC6YllGIZM/CSU-S4-Client-Reviews?type=design&node-id=370-32303&t=BbhPR4GCcHa4UpVA-0)

### Additional comments
In this PR, the headings sizes were overwritten.

### CODE REVIEW

File: src/components/02-components/cta-card/cta-card.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
+import { Button, Paper, Typography } from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+
+export type BrandProps = {
+  heading: string | null;
+  subHeading: string | null;
+  description: string | null;
```
Comment (cienvaras) [inline src/components/02-components/cta-card/cta-card.tsx:7]: This works, but if the prop optional is better this way:

```suggestion
  heading?: string;
  subHeading?: string;
  description?: string;
```

File: src/components/02-components/cta-card/cta-card.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  cta: string;
+  ctaTitle: string;
+};
+
+export default function CtaCard({ heading, subHeading, description, cta = '/', ctaTitle }: BrandProps) {
```
Comment (cienvaras) [inline src/components/02-components/cta-card/cta-card.tsx:12]: No need to provide a default value for the cta.

File: src/components/02-components/cta-card/cta-card.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+  const theme = useTheme();
+
+  // Styles
+  const headingStyles = {
+    fontWeight: '800',
```
Comment (cienvaras) [inline src/components/02-components/cta-card/cta-card.tsx:18]: Please change this weight to `700`, we're not using `800` in the project.

File: src/components/02-components/cta-card/cta-card.tsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+  const buttonStyles = {
+    backgroundColor: 'primary.main',
+    color: 'white',
+    fontWeight: '700',
+    '&:focus, &:hover, &:active': 
+      { backgroundColor: 'primary.dark' }
+  };
+
+  // Rendered
```
Comment (cienvaras) [inline src/components/02-components/cta-card/cta-card.tsx:39]: Make the comment more descriptive. Add a dot (`.`) at the end

**Review by cienvaras — CHANGES_REQUESTED** [review]
Looks great, just some minor fixes needed. Please check code comments.

Also, please adjust the button padding, to make it match the designs:

Screenshot 2023-05-12 at 11 16 21

**Comment by AnaBZC** [comment]
@cienvaras this is ready for re-review

**Review by cienvaras — APPROVED** [review]
🚀

**Pull request merged**